src/components/dialogs/load-creation-dialog.js

```diff
@@ -55,6 +55,7 @@ const LOAD_TYPES = [
  * @param workingNodeUuid : the node we are currently working on
```

Comment:
Add editData in a @param in the component documentation
Check if it's done for each dialog ?

Reply:
done